### PR TITLE
Add new links for more sources

### PR DESCRIPTION
**Pull request by VexingHex**
why did I do this

### CODE REVIEW

File: lib/Sagiri.js [hunk 1]
```diff
@@ -184,27 +187,32 @@ function resolveSauceURL(data) {
...
             // Pawoo
+            url = `https://pawoo.net/@${data.data.user_acct}/${data.data.pawoo_id};`
             break;
         default:
             throw new Error(`Unsupported site index: ${data.header.index_name.match(/^Index #(\d+):? /)[1]}`);
     }
 
     return url;
 }
+//SO...MUCH...PORN
```
Comment (sr229) [inline lib/Sagiri.js]: delet this

File: lib/Sagiri.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ class SauceHandler {
 
     /**
      * Gets the source and outputs it in your preferred output type
-     * 
+     *
```
Comment (sr229) [inline lib/Sagiri.js]: why the extra spaces?


clam down with whitespace choc

Comment (VexingHex) [reply lib/Sagiri.js]: its how atom imported it?

File: lib/Sagiri.js (hunk 1 above)
Comment (VexingHex) [reply lib/Sagiri.js]: no

**Comment by apap04** [comment]
why in hell are you using atom

**Comment by LaChocola** [comment]
I like it <3

**Comment by apap04** [comment]
then vscode pls

**Comment by apap04** [comment]
*use

**Comment by apap04** [comment]
just try it

**Pull request merged**